### ISSUE TITLE
Add attribute to advertise support of MULTIPOINT, MULTILINESTRING and MULTIPOLYGON

### ISSUE THREAD

**Issue by m-burgoyne** [opengeospatial/ogcapi-environmental-data-retrieval#323]
Supporting MULTIPOINT, MULTILINESTRING and MULTIPOLYGON could add a lot of complexity to an API server and are not always required.  Currently to only way to notify a user that MULTI queries are not supported is to return an error message, this could be resolved by adding an attribute to the `data_queries` section of the collections response.   With the default value (if the attribute is not defined)  being that MULTI queries are **NOT** supported.

i.e. 

```
      "data_queries": {
        "position": {
          "link": {
            "href": "http://www.test.demo/collections/demo/position",
            "hreflang": "en",
            "rel": "data",
            "variables": {
              "title": "Position query",
              "query_type": "position",
              "multipoint": "true",
              "output_formats": [
                "CoverageJSON",
                "GeoJSON",
                "XML"
              ],
              "default_output_format": "GeoJSON",
              "crs_details": [
                {
                  "crs": "CRS84",
                  "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
                }
              ]
            }
          }
        },
        "radius": {
          "link": {
            "href": "http://www.test.demo/collections/demo/radius",
            "hreflang": "en",
            "rel": "data",
            "variables": {
              "title": "Radius query",
              "description": "Radius query",
              "query_type": "radius",
              "multipoint": "false",
              "output_formats": [
                "CoverageJSON",
                "GeoJSON",
                "XML"
              ],
              "default_output_format": "GeoJSON",
              "within_units": [
                "km",
                "miles"
              ],
              "crs_details": [
                {
                  "crs": "CRS84",
                  "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
                }
              ]
            }
          }
        },
        "area": {
          "link": {
            "href": "http://www.test.demo/collections/demo/area",
            "hreflang": "en",
            "rel": "data",
            "variables": {
              "title": "Area query",
              "query_type": "area",
              "output_formats": [
                "CoverageJSON",
                "GeoJSON",
                "XML"
              ],
              "default_output_format": "CoverageJSON",
              "crs_details": [
                {
                  "crs": "CRS84",
                  "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
                }
              ]
            }
          },
          "trajectory": {
            "link": {
              "href": "http://www.test.demo/collections/demo/trajectory",
              "hreflang": "en",
              "rel": "data",
              "variables": {
                "title": "Area query",
                "query_type": "area",
                "multilinestring": "false",
                "output_formats": [
                  "CoverageJSON",
                  "GeoJSON",
                  "XML"
                ],
                "default_output_format": "CoverageJSON",
                "crs_details": [
                  {
                    "crs": "CRS84",
                    "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
                  }
                ]
              }
            }          
        }
```

**Comment by dblodgett-usgs** [comment]
Prefer to leave this out for now -- we have a lot of ambiguity in how these geometries get handled in the query patterns and I'd rather leave this generally open for implementers until we have more example implementations and find a need to start locking these things down.

**Comment by chris-little** [comment]
EDR API SWG 68 agrees this is a good idea, in that it reduces error messages, and enables very simple clients. Agreed to add to V1.1.

**Comment by ghobona** [comment]
There is an example of a MULTIPOLYGON query in Clause 8.2.4.1 of version 1.0.0.

Since MULTIPOLYGON support is going to be introduced in version 1.1, the example will need to be removed from v1.0.1 or replaced by a POLYGON one.

**Comment by m-burgoyne** [comment]
@ghobona MULTIPOINT, MULTILINESTRING and MULTIPOLYGON are supported in v1.0.1 of the specification, the aim of this 
 change is to allow a collection to advertise that it does not have support for MULTIPOINT, MULTILINESTRING and MULTIPOLYGON.

**Comment by m-burgoyne** [comment]
Thanks @ghobona, I have added a pull request with changes to fix the issues that you have spotted (#358)

**Comment by chris-little** [comment]
PR merged at EDR API SWG 75, 2022-04-28
